media_kit/lib/src/player/libmpv/core/initializer_isolate.dart

```diff
@@ -162,13 +194,26 @@ abstract class InitializerIsolate {
     while (true) {
       completer = Completer();
       final event = mpv.mpv_wait_event(handle, _isReleaseMode ? -1 : 0.1);
```

Comment:
On macOS, issue #180 is still effective, is the infinite timeout really necessary in release mode?

See https://github.com/alexmercerind/media_kit/issues/180#issuecomment-1542082084

```suggestion
      final event = mpv.mpv_wait_event(handle, 0.1);
```

Reply:
I'm not sure about this one.

It degrades performance quite a bit. I don't want to look bad from performance stand-point. In debug mode, we enable it to keep "hot-reload" functional (& optimal performance isn't a goal in debug builds anyway).

Maybe we can skip posting `MPV_EVENT_NONE` outside the `Isolate`.